src/Form/Input/Input.component.tsx

```diff
@@ -24,36 +25,30 @@ interface InputProps {
 }
 
 interface InputState {
-  isFocused: boolean;
+  isFocused?: boolean;
   value: string;
-  errorMessage: string;
+  errorMessage?: string;
 }
 
+const baseFontWeight = 500;
+
 export const StyledInput = styled.div `
-  width: calc(100% - 2rem);
-  height: 3rem;
-  position: relative;
   display: flex;
   flex-direction: column;
   font-family: ${fonts.fontFamily};
-  opacity: ${({disabled}: InputProps) => disabled && '0.5'};
 
-  .field-input {
-    width: 100%;
-    box-sizing : border-box;
-    padding: 24px 0 6px 0;
-    padding-left: ${({ prefix }: InputProps) => !prefix ? '.75rem' : 'calc(3rem - 3px)'};
-    border-radius: 2px;
-    border: solid 1px ${colors.neutrals.ash.light};
+  .input {
+    width: calc(100% - 2rem);
+    height: 3rem;
     position: relative;
-    font-size: calc(1rem - 2px);
-    font-weight: 500;
-    background: none;
-    z-index: 1;
+    display: flex;
+    flex-flow: row wrap;
+    border-radius: ${variables.borderRadius};
+    border: solid 1px ${colors.neutrals.ash.light};
+    opacity: ${({disabled}: InputProps) => disabled && '0.5'};
```

Comment:
This is something I need to change in the `tsconfig`, but can we be consistent with the space in destructuring of the props from an object. I prefer `({ disabled })` over `({disabled})`, but I'm open to feedback. Whatever we decide, I'll add to the tsconfig and prettier so we don't have to worry about this in the future.